src/account/AccountItem.js

```diff
@@ -26,34 +26,54 @@ const styles = createStyleSheet({
     marginLeft: 16,
   },
   text: {
-    color: BRAND_COLOR,
     fontWeight: 'bold',
     marginVertical: 2,
   },
   icon: {
     padding: 12,
     margin: 12,
   },
+  signOutText: {
+    fontStyle: 'italic',
+    fontSize: 12,
+    textAlign: 'right',
+    position: 'absolute',
+    end: 10,
+    top: 3,
+    color: 'rgba(0,0,0,.5)',
+  },
 });
 
 type Props = $ReadOnly<{|
-  index: number,
-  email: string,
-  realm: URL,
+  accounts: AccountStatus,
```

Comment:
Let's use a singular name (e.g., 'account') for a value that just represents one thing; the plural 'accounts' is a bit confusing.